pages/about.html

```diff
@@ -2,9 +2,41 @@
 <html>
 <head>
   <link rel="stylesheet" type="text/css" href="about.css">
+  <link rel="stylesheet" type="text/css" href="../index.css">
 </head>
 <body>
-  About us.
-  <footer></footer>
+  <section class="SCA-header">
+    <div class="SCA-header-title">
+      <span class="SCA-header-title-school"><b>Scoil Maelruain Senior</b></span>
+      <span class="SCA-header-title-app">School Connect App</span>
+    </div>
+    <ul class="SCA-header-menu">
+      <li class="SCA-header-menu-list"><a href="../index.html">Home</a></li>
+      <li class="SCA-header-menu-list">Talk To Us</li>
+      <li class="SCA-header-menu-list">News</li>
+      <li class="SCA-header-menu-list"><a href="./about.html"><u>About Us</u></a></li>
+    </ul>
+  </section>
+  <section class="SCA-about">
```

Comment:
? What is inside ?
You should have all content in the section as we discussed

Reply:
oh ok